camelot/view/controls/formview.py

```diff
@@ -388,10 +388,11 @@ def close_view( self, accept ):
         if is_deleted(self):
             return
         if (accept == True):
-            # clear mapping to prevent data being written again to the model,
-            # when the underlying object would be reverted
             form = self.findChild( QtWidgets.QWidget, 'form' )
             if form is not None:
+                form.submit()
```

Comment:
deze moet er in een volgende pr wel zeker terug uit heh ...

Reply:
Ja, is er hier al terug uit.